### PR TITLE
Add color adding feature

### PR DESCRIPTION
**Pull request by Jay-Gadhiya**
Test  link: https://cheer-notes-iu4nijhxw-jay-gadhiya.vercel.app/

functionality: 
1. add input with color
2. change color of the notes card
3. change color on edit note 

note: click on the color palette icon will open a color palette and close by again click on that, give me suggestions if you know how to close the color palette by clicking on any other place.

### CODE REVIEW

File: src/Pages/Home/Home.jsx [hunk 1]
```diff
@@ -32,6 +38,12 @@ const HomePage = () => {
             addNote(userNote, authState, noteDispatch, setUserNote)
            
     }
+
+    const colorPalateShow = (note) => {
+        setColorPalate(pre => !pre);
+    }
```
Comment (gopalbbharadva) [inline src/Pages/Home/Home.jsx:42]: Suggestion : It can be inliner

File: src/Pages/Home/Home.jsx [hunk 2]
```diff
@@ -41,44 +53,64 @@ const HomePage = () => {
...
+                                <button onClick={() => userColorInputHandler("red")} className="btn-color red"></button>
+                                <button onClick={() => userColorInputHandler("orange")} className="btn-color orange"></button>
+                                <button onClick={() => userColorInputHandler("yellow")} className="btn-color yellow"></button>
+                                <button onClick={() => userColorInputHandler("green")} className="btn-color green"></button>
+                                <button onClick={() => userColorInputHandler("teal")} className="btn-color teal"></button>
+                                <button onClick={() => userColorInputHandler("purple")} className="btn-color purple"></button>
+                                <button onClick={() => userColorInputHandler("pink")} className="btn-color pink"></button>
+                                <button onClick={() => userColorInputHandler("brown")} className="btn-color brown"></button>
+                                <button onClick={() => userColorInputHandler("grey")} className="btn-color grey"></button>
```
Comment (gopalbbharadva) [inline src/Pages/Home/Home.jsx:103]: Suggestion: You can create colors array and map over it which will return button with current color and handler.

**Review by gopalbbharadva — APPROVED** [review]
functionality working fine , good to go.

File: src/Pages/Home/Home.jsx (hunk 2 above)
Comment (Jay-Gadhiya) [reply src/Pages/Home/Home.jsx:103]: i will implement that

**Review by jeetbhalani15 — APPROVED** [review]
-  All API calls Are working fine
- I like adding colors to the notes UI & with its working. great bro !!
- Readable code & you can use pointer-events  to close the color palette may be.

**Pull request merged**